chapters/rum/chapter1/10.mdx

```diff
@@ -0,0 +1,258 @@
+<!-- DISABLE-FRONTMATTER-SECTIONS -->
+
+# Quiz de sfârșit de capitol[[test-de-sfârșit-de-capitol]]
+
+<CourseFloatingBanner
+    chapter={1}
+    classNames="absolute z-10 right-0 top-0"
+/>
+
+Acest capitol a acoperit o mulțime de subiecte! Nu vă faceți griji dacă nu ați înțeles toate detaliile; capitolele următoare vă vor ajuta să înțelegeți cum funcționează lucrurile în mecanismele lor interne.
+
+Mai întâi, însă, să testăm ceea ce ați învățat în acest capitol!
+
+
+### 1. Explorați Hub-ul și căutați checkpoint-ul `roberta-large-mnli`. Ce sarcină îndeplinește acesta?
+
+
+<Question
+	choices={[
+		{
+			text: "Summarization",
```

Comment:
I think it'd be nicer to also translate the choices here and the explanation for each question